src/main/java/seedu/storemando/logic/parser/ReminderCommandParser.java

```diff
@@ -13,45 +13,91 @@
 
     public static final String DAY_KEYWORD = "day";
     public static final String WEEK_KEYWORD = "week";
+    public static final String DAYS_KEYWORD = "days";
+    public static final String WEEKS_KEYWORD = "weeks";
 
     /**
-     * Convert the given number and the time unit to the number of days.
+     * Parses the given {@code String} of arguments in the context of the ReminderCommand
+     * and returns a ReminderCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    @Override
+    public ReminderCommand parse(String args) throws ParseException {
+        try {
+            String trimmedArgs = args.trim();
+            if (trimmedArgs.isEmpty()) {
+                throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ReminderCommand.MESSAGE_USAGE));
+            }
+
+            String[] wordsInTrimmedArgs = trimmedArgs.replaceAll("\\s{2,}", " ").split(" ");
```

Comment:
Can you clarify how is the parsing is handled? Would this break if there is an additional space passed in?

Reply:
no, it basically takes in all the words in the args, so say "       3             day        ", it will become "3" and "days" :)